lib/presenters/terminal.rb

```diff
@@ -12,26 +12,16 @@ def initialize(args = {})
       end
 
       def present
-        ComponentsDiff::Runner.run(branch).each do |component, details|
+        to_write = []
+        ComponentsDiff::Runner.run(branch, release_type).each do |component, details|
           print_details component, details
           next if /core|runtime|api/.match?(component)
 
-          revision = details[:tag].match(/[0-9]+\.[0-9]+\.[0-9]+/)
-          add_row(component, revision.to_s, details[:commits].any?)
+          add_row(component, details, details[:commits].any?)
```

Comment:
No need to send `details[:commits].any?` as an extra parameter if you are already sending `details`